### ISSUE TITLE
`Out-ConsoleGridView` Prevents `Test-Connection` Running

### ISSUE THREAD

**Issue by chenrylee** [PowerShell/GraphicalTools#63]
I've tested it on both Windows 10 (19041.113) and MacOS (10.15.3):
- PowerShell 7.0 GA
- Microsoft.PowerShell.ConsoleGUITools 0.3.0

1. Run `Test-Connection 8.8.8.8`, it works; 
2. Run `gci | Out-ConsoleGridView`, close it and run `Test-Connection 8.8.8.8`, it doesn't work, `CTRL` + `C` can't terminate it;
3. Close the window directly, and re-open the terminal;
4. Run `Test-Connection 8.8.8.8`, it works; 
5. Run `gci | Out-ConsoleGridView`, close it and run `gci`, it works;
6. Run `Test-Connection 8.8.8.8`, it doesn't work, `CTRL` + `C` can't terminate it;

**Comment by iSazonov** [comment]
In https://github.com/PowerShell/PowerShell/pull/11517 we fix such issue with WinForms. It seems the fix is not in 7.0 GA.

**Comment by TylerLeonhardt** [comment]
I've tested this with the latest daily build of PowerShell and confirmed that it works now.

```
❯❯❯ gci | ocgv                                                                                                                                                                                                                                ❯❯❯ Test-Connection google.com

   Destination: google.com

Ping Source           Address                   Latency BufferSize Status
                                                   (ms)        (B)
---- ------           -------                   ------- ---------- ------
   1 Timmernation.re… 172.217.14.206                  6         32 Success
   2 Timmernation.re… 172.217.14.206                 12         32 Success
```

There's nothing we can do about this so I'll close it. 7.1.0-preview.1 should have the fix!

**Comment by chenrylee** [comment]
> I've tested this with the latest daily build of PowerShell and confirmed that it works now.
> 
> ```
> ❯❯❯ gci | ocgv                                                                                                                                                                                                                                ❯❯❯ Test-Connection google.com
> 
>    Destination: google.com
> 
> Ping Source           Address                   Latency BufferSize Status
>                                                    (ms)        (B)
> ---- ------           -------                   ------- ---------- ------
>    1 Timmernation.re… 172.217.14.206                  6         32 Success
>    2 Timmernation.re… 172.217.14.206                 12         32 Success
> ```
> 
> There's nothing we can do about this so I'll close it. 7.1.0-preview.1 should have the fix!

<br />

> This bug will permanently exist for PowerShell 7.0 stable release.  

Did I get the point? 😵

**Comment by TylerLeonhardt** [comment]
hmm not sure. @SteveL-MSFT is this something worthy of servicing in a 7.0.1?

**Comment by iSazonov** [comment]
I think the fix is important to have in 7.0 servicing because users us WinForms. Please vote in referenced PR.

**Comment by chenrylee** [comment]
> hmm not sure. @SteveL-MSFT is this something worthy of servicing in a 7.0.1?

It was announced that it's a LTS release, so I think this bug will be fixed somehow in the coming releases. 🤔
